src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -743,6 +744,13 @@ private async Task ValidateSignUpPoliciesAsync(Guid ownerId)
             try
             {
                 await _organizationRepository.CreateAsync(organization);
+                await _organizationApiKeyRepository.CreateAsync(new OrganizationApiKey
```

Comment:
I don't think that is a great idea. It would also have to added to the ProviderOrganizationController and I think this is the right encapsulation of this logic. Before an ApiKey would be created right here directly on the table and now it's still created right here just it so happens to be on another table.

Reply:
you're right, leave as is